cmd/k8s-rdma-shared-dp/main.go

```diff
@@ -47,6 +47,10 @@ func main() {
 		log.Fatalf("Exiting.. one or more invalid configuration(s) given: %v", err)
 	}
 
+	if err := rm.EnsureRdmaSystemMode(); err != nil {
```

Comment:
older kernel (<5.3) will not support rdma system command
these systems are by default in shared mode.  we should not fail in this case

Reply:
It works with 5.2. According to release notes [1] Ubuntu 20.04 uses 5.4


https://wiki.ubuntu.com/FocalFossa/ReleaseNotes#Linux_Kernel